app/controllers/training_modules_users_controller.rb

```diff
@@ -29,6 +29,14 @@ def mark_exercise_complete
     render 'courses/_block', locals: { block:, course: block.course }
   end
 
+  def find_slide
+    @dashboard_url = 'https://outreachdashboard.wmflabs.org/'
```

Comment:
There's no need to make this an absolute link, and it shouldn't be hard-coded to `outreachdashboard.wmflabs.org` (as this feature will also be on `dashboard.wikiedu.org`).